### DISCUSSION TITLE
Couldn't find a compatible Webview2 Runtime installation to host WebViews.

### DISCUSSION THREAD

**Discussion by sangeetha0708** [MicrosoftEdge/WebView2Feedback#4331] [Q&A]
I have Fixed Weview Runtime Microsoft.WebView2.FixedVersionRuntime.120.0.2210.144.x64 and package installed as<package id="Microsoft.Web.WebView2" version="1.0.2210.55" targetFramework="net45" />

and i have followed the same steps in Webview Documentation  but it shows Couldn't find a compatible Webview2 Runtime installation to host WebViews. Kindly help anyone to resolve this issue

**Comment by victorhuangwq** [comment]
Can you try installing the Fixed Runtime as an administrator?
